src/components/PopoverMenu.tsx

```diff
@@ -35,6 +37,9 @@ type PopoverMenuItem = MenuItemProps & {
 
     /** Determines whether the menu item is disabled or not */
     disabled?: boolean;
+
+    /** Determines whether the menu item's onSelected() function is called after modal hide or not */
```

Comment:
I'd mention the scenario it was meant to fix in more detail - that opening one modal after another was not working without that